packit.yaml

```diff
@@ -0,0 +1,34 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: fedora-messaging.spec
+
+actions:
+    post-upstream-clone: "wget https://src.fedoraproject.org/fork/t0xic0der/rpms/fedora-messaging/raw/f35-300-src/f/fedora-messaging.spec -O fedora-messaging.spec"
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    metadata:
+      targets:
+        - fedora-35
+      branch: packit
+      owner: t0xic0der
+      project: fedora-messaging
+  - job: copr_build
+    trigger: release
+    metadata:
+      targets:
+        - fedora-35
+      branch: packit
+      owner: t0xic0der
+      project: fedora-messaging
+
+# add or remove files that should be synced
+files_to_sync:
+    - fedora-messaging.spec
+    - packit.yaml
```

Comment:
I can't remember if we agreed on `.packit.yml` or `packit.yml`; but either way, we should all use the same.

Reply:
We agreed on `.packit.yml` so yeah, that needs to be changed. Thanks for marking it out.